server/helpers/utils.js

```diff
@@ -94,6 +94,17 @@ const subscriptionSave = async data => {
 
   const { address, chatId } = data
 
+  const { getLivepeerDelegatorAccount } = require('./livepeerAPI')
+  const delegatorAccount = await getLivepeerDelegatorAccount(address)
+
+  if (delegatorAccount.status !== 'Bonded') {
```

Comment:
Could you delegatorAccount && delegatorAcount.status check ?, thanks

Reply:
Done